frontend/integration-tests/tests/kubevirt/utils/utils.ts

```diff
@@ -117,6 +117,22 @@ export async function logIn() {
   await browser.wait(until.visibilityOf($('img.pf-c-brand')), PAGE_LOAD_TIMEOUT);
 }
 
+export function getRandStr(length: number) {
+  return Math.random().toString(36).replace(/[.]/g, '').substr(1, length); // First char is always 0;
+}
+
+export function getRandomMacAddress() {
+  const getRandByte = () => {
+    let byte: string;
+    do {
+      byte = Math.random().toString(36).replace(/[.g-z]/g, '').substr(2, 2);
```

Comment:
we just need a random hex number right? Why not just `Math.random().toString(16)` ?

Reply:
Yes, thanks. Updated